core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy

```diff
@@ -0,0 +1,51 @@
+package jenkins.model.GlobalCloudConfiguration
+
+import hudson.slaves.Cloud
+
+
+def f = namespace(lib.FormTagLib)
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
+    l.side_panel {
+        l.tasks {
+            l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
+            l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/computer/", title:_("Manage Nodes"))
+        }
+    }
+    l.main_panel {
+        h1 {
+            l.icon(class: 'icon-health-40to59 icon-xlg')
+            // TODO more appropriate icon
```

Comment:
does this belong one line up? i.e. above the icon.
and is it still relevant?

Reply:
I don't know. `icon-health-40to59` looks like a hack to me. Personally I find it funny (it's a cloud), but wouldn't be surprised if others are confused by this icon.

(TBH, I just it copied from the tool configuration view, but it's oddly applicable here as well, so can be left in.)

Re the exact positioning of the TODO comment while the context is clear, I think it doesn't matter.